sktime/datatypes/_series/_check.py

```diff
@@ -545,6 +545,48 @@ def _index_equally_spaced(index):
 
 class SeriesXarray(ScitypeSeries):
     """Data type: xarray based specification of single time series.
+    
+    Name: ``"xr.DataArray"``
+
+    Short description:
+    
+    An `xarray.DataArray` representing a time series, where:
```

Comment:
please use double backticks to ensure code blocks

Reply:
I've done that as well in the new updated commit